NovelRT.Sdk/TemplateFiles/CMakeTemplate/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-﻿cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
+﻿cmake_minimum_required(VERSION 3.18 FATAL_ERROR)

```

Comment:
With testing this could probably go lower - if the engine is already built and not being paired with the application, then it's not solely dependent on 3.19.